pkg/services/ngalert/provisioning/alert_rules.go

```diff
@@ -87,6 +86,45 @@ func (service *AlertRuleService) CreateAlertRule(ctx context.Context, rule model
 	return rule, nil
 }
 
+// UpdateRuleGroup will update the interval for all rules in the group.
+func (service *AlertRuleService) UpdateRuleGroup(ctx context.Context, orgID int64, namespaceUID string, ruleGroup string, interval int64) error {
+	if err := service.validateRuleGroupInterval(interval); err != nil {
+		return err
+	}
+	return service.xact.InTransaction(ctx, func(ctx context.Context) error {
+		query := &models.ListAlertRulesQuery{
+			OrgID:         orgID,
+			NamespaceUIDs: []string{namespaceUID},
+			RuleGroup:     ruleGroup,
+		}
+		err := service.ruleStore.ListAlertRules(ctx, query)
+		if err != nil {
+			return err
```

Comment:
Could we add `fmt.Errorf` to add some more context to this error?

```suggestion
			return fmt.Errorf("failed to list alert rules: %w", err)
```

Reply:
Good shout, will add it.